Tools/WinMLRunner/src/OutputHelper.h

```diff
@@ -518,16 +540,18 @@ class OutputHelper
     }
     void SaveBindTimes(Profiler<WINML_MODEL_TEST_PERF>& profiler, uint32_t iterNum)
     {
-        m_clockBindTimes[iterNum] = profiler[BIND_VALUE].GetClockTime();
+        m_clockBindTimes[iterNum] =
+            (iterNum == 0) ? profiler[BIND_VALUE_FIRST_RUN].GetClockTime() : profiler[BIND_VALUE].GetClockTime();
     }
     void SaveEvalPerformance(Profiler<WINML_MODEL_TEST_PERF>& profiler, uint32_t iterNum)
     {
-        m_clockEvalTimes[iterNum] = profiler[EVAL_MODEL].GetClockTime();
-        m_CPUWorkingDiff[iterNum] = profiler[EVAL_MODEL].GetCpuWorkingDiff();
-        m_CPUWorkingStart[iterNum] = profiler[EVAL_MODEL].GetCpuWorkingStart();
-        m_GPUSharedDiff[iterNum] = profiler[EVAL_MODEL].GetGpuSharedDiff();
-        m_GPUSharedStart[iterNum] = profiler[EVAL_MODEL].GetGpuSharedStart();
-        m_GPUDedicatedDiff[iterNum] = profiler[EVAL_MODEL].GetGpuDedicatedDiff();
+        iterationNum = (iterNum == 0) ? EVAL_MODEL_FIRST_RUN : EVAL_MODEL;
```

Comment:
Any reason why iterationNum is being used like this here as an index to access the performance arrays (m_clockEvalTimes, m_CPUWorkingDiff, m_CPUWorkingStart, etc, etc)? It will always either be EVAL_MODEL_FIRST_RUN (5) or EVAL_MODEL (6)..